classes/battle/Fighter.php

```diff
@@ -10,7 +10,7 @@ abstract class Fighter {
     const BLOODLINE_OFFENSE_RATIO = self::SKILL_OFFENSE_RATIO * 0.8;
     const BLOODLINE_DEFENSE_MULTIPLIER = 35;
 
-    const SPEED_OFFENSE_RATIO = 0.25;
+    const SPEED_OFFENSE_RATIO = 0.35;
```

Comment:
What's the rationale on this one?

Reply:
Looking at the balance of high builds relative to what they're supposed to counter and what counters them.

The most common two types of builds are 0 speed and ~85k speed. These two types of builds are roughly balanced versus each other - the same bloodline at 0 and 85k speed will deal almost identical damage to each other. 

A build with speed beyond 85k should expect to gain advantage versus 85k while becoming weaker against builds closer to 0 speed. The issue is the tradeoff, where speed beyond 85k is only marginally helpful versus other builds while being substantially weaker versus 0. 

So you have a case where high speed (like >110k) is an automatic loss to 0 speed without having a notable advantage versus ~85k. 

The proposed change is very small in terms of overall power but I think is enough to make speed worthwhile:
```Adjusted Speed offense ratio from 0.25 => 0.35
Calculates to a 10.2% increase in damage output for a 50k BL / 50k Offense / 150k Speed build
Calculates to a 3.9% increase in damage output for a 85k BL / 85k Offense / 80k Speed build```

Reply:
Makes sense, let's do it 👍